urdf/src/model.cpp

```diff
@@ -34,23 +34,47 @@
 
 /* Author: Wim Meeussen */
 
+#include "urdf/model.h"
+#include <urdf_parser_plugin/parser.h>
+#include <pluginlib/class_loader.hpp>
+
+#include <cassert>
 #include <fstream>
 #include <iostream>
+#include <limits>
 #include <string>
+#include <utility>
 #include <vector>
 
-#include "urdf/model.h"
-
-/* we include the default parser for plain URDF files;
-   other parsers are loaded via plugins (if available) */
-#include "urdf_parser/urdf_parser.h"
 
 namespace urdf
 {
+class ModelImplementation
```

Comment:
Nit: this can be a `final` class, since nothing will be deriving from it.

Reply:
Final in 06474e8abdd6befc2c508f13561c290662a3b4cf